api/v01/entities/vrp_result.rb

```diff
@@ -32,14 +32,11 @@ class VrpResultDetailQuantities < Grape::Entity
     end
 
     class VrpResultSolutionRouteActivityDetails < Grape::Entity
-      expose :router_mode, documentation: { type: String, desc: 'Means of transport used to reach this activity, it may vary within a route if subtours are defined' }
```

Comment:
These fields are usefull in the case of multi_modal routes : https://github.com/Mapotempo/optimizer-api/blob/c3e0b1732e85c95e21c0bf8982cc0b4f1704a1e5/lib/interpreters/multi_modal.rb#L152

Reply:
Shouldn't this rather be in VrpResultSolutionRoute ?

Reply:
In the multi modal case, the router_mode can change within the same route

Reply:
Right, I will fix this then !